Python/ceval.c

```diff
@@ -1675,10 +1675,10 @@ static void
 clear_thread_frame(PyThreadState *tstate, _PyInterpreterFrame * frame)
 {
     assert(frame->owner == FRAME_OWNED_BY_THREAD);
-    // Make sure that this is, indeed, the top frame. We can't check this in
-    // _PyThreadState_PopFrame, since f_code is already cleared at that point:
-    assert((PyObject **)frame + _PyFrame_GetCode(frame)->co_framesize ==
-        tstate->datastack_top);
+    // // Make sure that this is, indeed, the top frame. We can't check this in
+    // // _PyThreadState_PopFrame, since f_code is already cleared at that point:
+    // assert((PyObject **)frame + _PyFrame_GetCode(frame)->co_framesize ==
+    //     tstate->datastack_top);
```

Comment:
Obviously this block should just be deleted. I'm leaving it in because it seems a kind of important check, but I don't know how to reproduce it now that datastack_top may be larger than the end of the reserved stack space due to call inlining.